src/Pages/yeongseoblee/Login/Login.js

```diff
@@ -0,0 +1,165 @@
+      }),
+    })
+      .then((res) => res.json()) // JSON body => JS
+      .then((res) => sessionStorage.setItem("access_token", res.access_token));
+  };
+
+  handleChange = ({ target: { name, value } }) => {
+    this.setState({ [name]: value });
+  };
+
+  render() {
+    const screenshots = [
+      "https://www.instagram.com/static/images/homepage/screenshot1-2x.jpg/9144d6673849.jpg",
+      "https://www.instagram.com/static/images/homepage/screenshot2-2x.jpg/177140221987.jpg",
+      "https://www.instagram.com/static/images/homepage/screenshot3-2x.jpg/ff2c097a681e.jpg",
+      "https://www.instagram.com/static/images/homepage/screenshot4-2x.jpg/b27a108592d8.jpg",
+      "https://www.instagram.com/static/images/homepage/screenshot5-2x.jpg/5e04169b9308.jpg",
+    ];
+    const { account, password } = this.state;
+
+    const footerObj = {
+      about: "https://about.instagram.com/",
+      help: "https://help.instagram.com/",
+      press: "https://about.instagram.com/blog/",
+      api: "https://www.instagram.com/developer/",
+      jobs: "https://www.instagram.com/about/jobs/",
+      privacy: "https://help.instagram.com/519522125107875",
+      terms: "https://help.instagram.com/581066165581870",
+      locations: "https://www.instagram.com/explore/locations/",
+      "top accounts": "https://www.instagram.com/directory/profiles/",
+      hashtags: "https://www.instagram.com/directory/hashtags/",
+      language: "#",
+    };
+    return (
+      <div className="Login">
+        <article>
+          <div className="leftContainer">
+            <div className="phoneFrame">
+              <div className="imagesInPhone">
+                {screenshots.map((screenshotSrc, idx) => (
+                  <img key={idx} alt={`screensho${idx}`} src={screenshotSrc} />
+                ))}
+              </div>
+            </div>
+          </div>
+          <div className="rightContainer">
+            <div className="logInWrapper">
+              <h1 className="instaLogo">Instagram</h1>
+              <div className="logInInputsContainer">
+                <form>
+                  <div className="logInInputs" onChange={this.handleChange}>
+                    <input
+                      className="accountInput"
+                      type="text"
+                      name="account"
+                      babo="me"
+                      placeholder="Phone number, username, or email"
+                    />
+                    <input
+                      className="passwordInput"
+                      type="password"
+                      name="password"
+                      placeholder="Password"
+                      // value={password}
+                    />
+                    <button
+                      className={
+                        account.includes("@") && password.length > 5
+                          ? "logInBtnActive"
+                          : "logInBtn"
+                      }
```

Comment:
```suggestion
                          ? "logInBtn active"
                          : "logInBtn"
                      }
```
토글에 의해 변화하는 스타일 값이라면 이와 같이 띄어쓰기로 분리해서 쓰는 것이 좋습니다. 그렇게 하면 좋은 이유에 대해 댓글로 남겨주세요.